domains/pallets/domain-registry/src/lib.rs

```diff
@@ -0,0 +1,626 @@
+// Copyright (C) 2021 Subspace Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Domain Registry Module
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod tests;
+
+use frame_support::traits::{Currency, Get, LockIdentifier, LockableCurrency, WithdrawReasons};
+pub use pallet::*;
+use sp_domains::{BundleEquivocationProof, DomainId, FraudProof, InvalidTransactionProof};
```

Comment:
Okay, looks like we have two different domain_id types defined by both of us.  we should merge them.
Does sp_domains belongs under crates? I think it should be under domains/...
thoughts ?

Reply:
Yeah, we should unify the domain_id type, probably together with the domain classification according to domain_id as per your comment below.

Yes, `pallet-domains` on the primary chain is dependent on `sp-domains`, thus I think it makes sense to stay under crates/.

Reply:
but we can already pull `pallet-domains` under domains no? it is much more easier to have all the domain and execution under domain, IMO

Reply:
The current convention is that all crates used on the primary chain stay under `crates/`, so `pallet-domains` can't be pulled into `domains/`.